scale/ingest/test/triggers/test_ingest_recipe_handler.py

```diff
@@ -52,104 +53,121 @@ def setUp(self):
 
         self.recipe = recipe_test_utils.create_recipe_type_v6(name='test-recipe', definition=recipe_type_def)
 
+    
```

Comment:
Do we want to remove commented out code or reinstate?

Reply:
This should be re-instated. I was having issues with the build on this one (stuck due to a missing patch I think) and was going to come back to it once I got the others working.